packages/import-maps-generate/README.md

```diff
@@ -89,46 +117,56 @@ which will result in the following import map.
 }
 ```
 
+Overrides may be useful for:
+- Polyfilling
+- Fixing a dependency with a local fork
+- Getting a dependency from a CDN instead
+
+### Overriding Scopes
+
+You can also override entire scopes:
+
+```json
+{
+  "importMap": {
+    "overrides": {
+      "scopes": {
+        "lit-html/": "/path/to/lit-html/"
+      }
+    }
+  }
+}
+```
+
+
 ### Deletes
```

Comment:
`overrides` and `resolutions` are nouns, but `deletes` is a verb. Perhaps this key should be `deletions`?

Reply:
I agree, though this requires changes in the code as well so I propose to do it in a separate PR

Reply:
Actually nvm, I changed it in this PR